modules/get-started/pages/release-notes/operator.adoc

```diff
@@ -9,15 +9,44 @@ See also:
 * xref:upgrade:k-compatibility.adoc[]
 * xref:upgrade:k-rolling-upgrade.adoc[]
 
-== Redpanda Operator v25.1.x
+NOTE: Redpanda Console v3 is **not yet available when deploying with the Redpanda Operator**. The Redpanda Operator continues to deploy Redpanda Console v2. To try Redpanda Console v3 in Kubernetes, you can xref:deploy:deployment-option/self-hosted/kubernetes/k-production-deployment.adoc[deploy Redpanda using the Redpanda Helm chart] instead of the Redpanda Operator.
 
-link:https://github.com/redpanda-data/redpanda-operator/blob/release/v25.1.x/operator/CHANGELOG.md[Changelog^]
+== Redpanda Operator v25.2.x
+
+link:https://github.com/redpanda-data/redpanda-operator/blob/release/v25.2.x/operator/CHANGELOG.md[Changelog^]
+
+=== Cluster scope by default
+
+Starting in v25.2, the Redpanda Operator defaults to cluster scope instead of namespace scope. This change provides several benefits:
+
+* **Simplified management**: A single operator instance can manage multiple Redpanda clusters across different namespaces
+* **Reduced resource overhead**: No need to deploy separate operator instances for each namespace
+* **Centralized upgrades**: Upgrade the operator once to benefit all managed Redpanda clusters
+* **Cross-namespace management**: Deploy the operator in a dedicated namespace (such as `redpanda-system`) while managing clusters in application namespaces
+* **Simplified RBAC for debug bundles**: The Redpanda Operator now provides all required permissions for `rpk` debug bundle collection by default. The `rbac.createRPKBundleCRs` flag is no longer needed
```

Comment:
I know you are following the convention that this bulleted list does not have end punctuation, but it's odd that this last one does not have a period when it comes right after a sentence with end punctuation. Maybe give them all periods and then they are consistent.